tests/common.go

```diff
@@ -3740,10 +3747,11 @@ func CreateAzureBucket(bucketName string) {
 }
 
 // createAzureBackupLocation creates backup location
-func createAzureBackupLocation(name, uid, cloudCred, cloudCredUID, bucketName, orgID string) {
-	Step(fmt.Sprintf("Create Azure backup location [%s] in org [%s]", name, orgID), func() {
-		CreateAzureBackupLocation(name, uid, cloudCred, cloudCredUID, bucketName, orgID)
-	})
+func createAzureBackupLocation(name, uid, cloudCred, cloudCredUID, bucketName, orgID string) error {
```

Comment:
Why do you have this function, that all it does is calls the actual function `CreateAzureBackupLocation()` that does the actual backup creating on Azure, this seems redundant, can we remove it and just call the `CreateAzureBackupLocation()` instead where you need to?

Reply:
Yes Nikolay these are the things we will be addressing in future PRs.

Reply:
You are touching this whole function in this PR, why not just remove it and use `CreateS3BackupLocation()` instead?

Reply:
done